cg/constants/orderforms.py

```diff
@@ -28,6 +28,7 @@ class Orderform(StrEnum):
     RNAFUSION: str = "1508"
     RML: str = "1604"
     SARS_COV_2: str = "2184"
+    MICROBIAL_FASTQ: str = "microbial_sequencing"
```

Comment:
Isn't there a way to assign also a number to this? Unclear how this works but it seems to me that the new orderform should be standardised with the previous convention

Reply:
The numbers are basically from our old document-system that had an ID for each document. Since we no longer use it I don't see a point in making up a new number. @islean Do you have an opinion?

Reply:
I much prefer naming it this way, although I wonder if this will make customer's try this order form for microsalt as well? 
<img width="978" alt="image" src="https://github.com/user-attachments/assets/7527e840-2ba9-4a36-9ad0-23beb45922a0">